### PR TITLE
DROTH-2273 (US 1891) Suggested assets-button is missing in municipali…

### PR DESCRIPTION
**Pull request by FilPinto**
…ty management page

-Fixed problem when clicking suggested asset button and it gave an error in server (500)
-Fixed expired suggested assets appearing on the list
-Fixed problem when clicking suggested asset button and no assets are shown

### CODE REVIEW

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/VerificationDao.scala [hunk 1]
```diff
@@ -115,6 +115,7 @@ class VerificationDao {
         join property p on p.asset_type_id = a.asset_type_id and p.property_type = 'checkbox' and p.public_id = 'suggest_box'
         join multiple_choice_value mcv on mcv.asset_id = a.id and mcv.property_id = p.id
         join enumerated_value ev on ev.property_id = p.id and ev.value = 1 and ev.id = mcv.enumerated_value_id and  lrm.link_id in (select id from #$idTableName)
```
Comment (ricardofmsimoes) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/VerificationDao.scala]: This part " and  lrm.link_id in (select id from #$idTableName)"
Shouldn't be at WHERE clause ?

Comment (FilPinto) [reply digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/VerificationDao.scala]: I left it as it was. No problem when executing the query.

Comment (marquesfm) [reply digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/VerificationDao.scala]: Why not at line 114?
`scala

join lrm_position lrm on lrm.id = al.position_id and lrm.link_id in (select id from #$idTableName)
`

Comment (marquesfm) [reply digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/VerificationDao.scala]: Let's keep it like it is, it's working and already has reverted to different ways two times and the final result was always this.

**Pull request merged**